### ISSUE TITLE
After some time, all saves on some android devices disappear

### ISSUE THREAD

**Issue by NoFanru** [renpy/renpy#4771]
People often write to me that they left the game, entered the game after a while and all saves disappeared (save slots, persistents variables, dialog history). Most often, the problem occurs on android 12 and 13.

I didn't notice this issue on my phone. I'm running Ren'Py 8.1.0, previously 8.0, had the same issue.

**Comment by the66F95** [comment]
most likely they run into the normal Android behavior (i think since Android 11) of revoking granted permissions and deleting the app's data if the app isn't used for a long time.
ask your users to toggle that setting of your app.

**Comment by NoFanru** [comment]
> скорее всего, они сталкиваются с нормальным поведением Android (я думаю, начиная с Android 11) отзыва предоставленных разрешений и удаления данных приложения, если приложение не используется в течение длительного времени. попросите своих пользователей переключить этот параметр вашего приложения.

Thanks for the answer!
They say it usually takes a day or less for data to go missing.

**Comment by renpytom** [comment]
I don't think this is something normal android devices do. I'd suggest filing an issue with the device's maker and seeing if they have any ideas.
